activitypub.go

```diff
@@ -133,12 +120,6 @@ func handleFetchCollectionOutbox(app *App, w http.ResponseWriter, r *http.Reques
 	}
 	c.hostName = app.cfg.App.Host
 
-	if app.cfg.App.SingleUser {
```

Comment:
I hesitated to remove this, but IMO if the `alias != c.Alias` this should be a bug. It seems in this function, because it is using the activity pub outbox, the alias of the collection must be set so I think it is safe to treat this how multi user applications treat collections in other areas of the code